src/PositionManager.sol

```diff
@@ -327,6 +327,8 @@ contract PositionManager is
             (liquidityDelta - feesAccrued).validateMinOut(amount0Min, amount1Min);
         }
 
+        if (positionConfigs[tokenId].hasSubscriber()) _unsubscribe(tokenId, config);
```

Comment:
Is there some reason this can't be done after the storage writes below to follow CEI? E.g. state the subscriber might need to read?

Reply:
on main, both modifyLiquidity + unsubscribe should adhere to CEI now

https://github.com/Uniswap/v4-periphery/blob/main/src/PositionManager.sol#L314-L327